logger/logger_test.go

```diff
@@ -223,3 +223,19 @@ func TestAddContext(t *testing.T) {
 	assertLogFormatAndCompareContent(t, string(buf.Bytes()),
 		kv.FormatLog("logger-tester", kv.Info, "2", M{"a": "b"}))
 }
+
+func TestFailAddReservedContext(t *testing.T) {
+	logger := New("logger-tester")
+	reservedKeyNames := map[string]bool{
+		"title":  true,
+		"source": true,
+		"value":  true,
+		"type":   true,
+		"level":  true,
+	}
+	for k := range reservedKeyNames {
+		res := updateContextMapIfNotReserved(logger.globals, k, "test")
+		msg := "Should not be able to set key " + k
+		assert.False(t, res, msg)
```

Comment:
yea, i debated it. i don't like the println side effect that was in there already, so i wanted to pass back a value. the test is better with checking the internals itself though. i'll keep it simple